Source/WebKit/UIProcess/API/APIFrameInfo.h

```diff
@@ -61,6 +61,8 @@ class FrameInfo final : public ObjectImpl<Object::Type::FrameInfo> {
     bool errorOccurred() const { return m_data.errorOccurred; }
     WTF::String title() const;
 
+    const WebKit::FrameInfoData& frameInfoData() const { return m_data; }
```

Comment:
We could alternatively make an accessor for the frame metrics.  That would be a little more consistent.  This is also fine.